### PR TITLE
Update README.md for building with eBPF support, and give a quick example for running rezolus

### PR DESCRIPTION
**Pull request by jmeichle**
Problem

Quick doc update for how to compile with eBPF support, and how to run and view metrics.

Solution

Figured out how to do this on my laptop, and updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,22 @@ cargo build --release && \
...
+1. Ensure that [bcc toolchain](https://github.com/iovisor/bcc/blob/master/INSTALL.md) is installed.
+1. Follow the previously listed steps for building, but include `--features ebpf` when calling `cargo build`.
+1. When running, you will require sudo for eBPF to work. Depending on your cargo setup it might be easier to build a binary and run that with `sudo`.
+
+### Viewing metrics from the example config
+
+Once rezolus is running, you can view the metrics via curl and `jq` for parsing the JSON:
+
+```bash
+curl http://localhost:4242/get | jq .
```
Comment (jmeichle) [inline README.md]: `/get` is semi arbitrary, but i t appears any route gives all metrics from `METRICS.md`

Comment (brayniac) [reply README.md]: Can you change this to `/vars` which gives human readable output natively?

https://github.com/twitter/rezolus/blob/master/src/stats/http.rs#L68

**Comment by brayniac** [comment]
Thanks for the PR! I need to think about some of the text a little bit more. I feel like we may want to be a little more explicit about some of the details. I'll give it some thought, but may opt to open a PR myself to cover the details that I'm thinking about. I'll leave this one open while I decide the best approach to take with this.

File: README.md (hunk 1 above)
Comment (jmeichle) [reply README.md]: Done!

**Comment by jmeichle** [comment]
@brayniac Totally understood about phrasing and doc structure. I just wanted to put something up since we had discussed at QCon some docs gaps around building with eBPF support. Otherwise, this seems pretty sweet.

**Comment by brayniac** [comment]
@jmeichle - take a look at #78 and let me know if we can close this one out in favor of that one. Don't mean to snipe your PR, but there were other specifics I wanted to capture that may have been non-obvious. 

Thanks again for opening this one. I appreciate that you took the time to work through that aspect of the build and document it.

**Pull request closed without merging**